src/components/Articles.jsx

```diff
@@ -1,23 +1,30 @@
 import React, { useEffect, useState } from "react";
 import { getArticles } from "../utils/api";
 import { Link } from "react-router-dom";
+import sortFunc from "../utils/helper-funcs";
 
 const Articles = () => {
   const [articles, setArticles] = useState([]);
-
   const [sortBy, setSortBy] = useState(undefined);
-
   const [order, setOrder] = useState(undefined);
-
   const [seeOnly, setSeeOnly] = useState("");
+  const [byVotes, setByVotes] = useState(false);
+  const [err, setErr] = useState(null);
 
   useEffect(() => {
-    getArticles(sortBy, order, seeOnly).then((data) => {
-      setArticles(data.articles);
-      console.log(articles);
-    });
-  }, [sortBy, order, seeOnly]);
-
+    getArticles(sortBy, order, seeOnly)
+      .then((data) => {
+        if (byVotes) {
+          let sorted = sortFunc(data.articles, order);
+          setArticles(sorted);
```

Comment:
same as above - using a helper function is a nice workaround in this case, but would be worthy of a refactor at a later point 🙂